### Basecalling Sanger Sequences

Today we are going to be "basecalling" (quality checking and trimming) your PCR products that have been sequenced on a Sanger Sequencer here at UC Riverside. First, we'll watch [this video about the Sanger sequencing technology](https://www.youtube.com/watch?v=e2G5zx-OJIw) and what has happened on the machine to produce your DNA sequences.

Next, we'll jump right into working with your DNA seqeunces:

1. First, you'll want to download a trial version of the **Sequencher** software onto your computer. This is as special program we'll use to view and trim our DNA sequences. [Click here for download link](https://www.genecodes.com/free-download) (you may be prompted to enter an email address, in which case use my email - holly dot bik at ucr dot edu)

2. Next, choose a worm from this class folder: [https://www.dropbox.com/sh/16hc2gsb2qlknfu/AAAn1KquFZjSEQ_nNJ37L45ea?dl=0](https://www.dropbox.com/sh/16hc2gsb2qlknfu/AAAn1KquFZjSEQ_nNJ37L45ea?dl=0) - we have enough DNA sequences for each person to have their own worm. We will go around and check off who is assigned to work with with each worm (no two people should be working with the same worm number).

3. Go into the worm subfolder (e.g. **Nem4** folder) and load the **.abi** files into the Sequencher program. These are the raw DNA sequence files. If correctly loaded, you will see colored peaks in Sequencer when you click around.

4. Assemble the sequences together. These are forward and reverse seqeunces from one PCR primer set.

5. Trim and basecall your sequences as we discuss in class. "Bad" bases should be removed from both ends, the primer sequences should also be removed, and any bases that disagree between fragments (bases marked with a black dot) should be manually inspected and corrected.

6. Once you are finished, export the clean DNA sequences into a FASTA text file from sequencher.

7. Open this text file and copy-paste the DNA sequence into [the NCBI online BLAST tool](https://blast.ncbi.nlm.nih.gov/Blast.cgi
)

8. Look through the BLAST results. What nematode species (or genus or family) does your DNA sequence match to?

9. Search online, and use the [Warwick & Platt pictorial key](https://www.dropbox.com/s/gu5pt0xn3nlpsa9/NematodePictorialKey_WarwickPlatt.pdf?dl=0) to find the corresponding morphology of your nematode worm. What mouthparts does it have? What amphid shape? Any cuticle striations or any other interesting morphology?

